src/main/java/net/mcreator/element/ModElementType.java

```diff
@@ -164,4 +165,7 @@ public boolean hasProcedureTriggers() {
 	public static ModElementType<?> TAG;
 	public static ModElementType<?> TOOL;
 
+
+	// Legacy

```

Comment:
Isn't one empty line enough?
```suggestion
	// Legacy
```